lib/designator/subject_set_iterators/randomly.ex

```diff
@@ -1,8 +1,8 @@
-defmodule Designator.RandomStream do
+defmodule Designator.SubjectSetIterators.Randomly do
```

Comment:
@camallen  Sounds good. I'm good with making this change. I'm wondering if there could be potential future confusion since there is `Designator.Random` module in place already, but it may not be too big of a deal since one is under `Designator.SubjectSetIterators`.